src/components/ReleaseNotes/ReleaseNotes.js

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import {
+  CardGroup,
+  Card,
+  CardHeader,
+  CardBody,
+  CardFooter,
+  Grid,
+  GridContainer,
+} from "@trussworks/react-uswds";
+import { constants } from "../../helpers/constants";
+import { NotFoundPage } from "../NotFoundPage/NotFoundPage";
+
+export const ReleaseNotes = () => {
```

Comment:
So, I think we don't need to do this fetch anymore. If you look at the new commit I made to the API docs page, you'll see that I just grabbed everything from the constants file.